apps/meteor/app/api/server/v1/oauthapps.ts

```diff
@@ -43,6 +43,40 @@ API.v1.addRoute(
 	},
 );
 
+API.v1.addRoute(
+	'oauth-apps.update',
+	{
+		authRequired: true,
+		validateParams: isUpdateOAuthAppParams,
+	},
+	{
+		async post() {
+			const { appId } = this.bodyParams;
+
+			const result = Meteor.call('updateOAuthApp', appId, this.bodyParams);
+
+			return API.v1.success(result);
+		},
+	},
+);
+
+API.v1.addRoute(
+	'oauth-apps.delete',
+	{
+		authRequired: true,
+		validateParams: isDeleteOAuthAppParams,
+	},
+	{
+		async delete() {
```

Comment:
```suggestion
		async post() {
```
I guess this should now be a POST request